### ISSUE TITLE
Validate field-level CMS entity transformation

### ISSUE THREAD

**Issue by cvalarida** [department-of-veterans-affairs/va.gov-team#10449]
## Issue Description
In an effort to automate the entity transformation for using the content from CMS export in the Liquid templates, we need to prototype / validate an automated approach. The idea is that this would replace the manual creation of transformers for each bundle (what we've been calling a "[content model](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/cms-integration/transformation-process.md#content-model)")

The current approach is to have a transformer function for each bundle. The approach I'm prototyping in this ticket is to transform a bundle using a system which iterates over each field in an entity's transformed schema and finds a field transformer function based on the input and output schemas.

If successful, this will enable us to do https://github.com/department-of-veterans-affairs/va.gov-team/issues/10417 to automate the creation of the input schema.

## Acceptance Criteria
- [ ] A new bundle type is transformed using this approach
- [ ] An existing transformer is replaced by this approach
- [ ] The transformation of all other bundles are unaffected (for now)

**Comment by cvalarida** [comment]
**Update:**
This was going well until I ran into a problem with the output schemas having circular references.

Specifically, the issue was that any paragraph could have a child entity of any other kind of paragraph. Previuosly, we were using the JSON schema `$ref`, but with the new approach, in order to resolve circular references in the _data_, we expanded the `$ref`s into full schemas (so we could exclude the property causing the circular reference). The trouble with that now is with Node's `require()`:
1. We `require('./schemas/transformed/paragraph-whatever')`
1. That module has a `require('./schemas/helpers')` to get the schema for all paragraphs
    - `anyOf: [ { /* paragraph-alert */ }, { ... } ]`
1. To compile the list of all paragraph schemas, the `schema/helpers.js` module `require()`s `paragraph-whatever.js`, which then returns `{}` because it's a circular dependency.

I toyed with the idea of using `$ref`s and expanding them at runtime (rather than using `require()`), but haven't explored it yet. We're quickly getting pretty complicated. :disappointed:
